<ui version="4.0" >
 <class>WdgNewImage</class>
 <widget class="QWidget" name="WdgNewImage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>New Image</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="lblName" >
     <property name="text" >
      <string>&amp;Name:</string>
     </property>
     <property name="buddy" >
      <cstring>txtName</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtName" >
     <property name="text" >
      <string>untitled-1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpImage" >
     <property name="title" >
      <string>Image Size</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="lblWidth" >
          <property name="text" >
           <string>Width:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblHeight" >
          <property name="text" >
           <string>Height:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring></cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QDoubleSpinBox" name="doubleWidth" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="decimals" >
           <number>2</number>
          </property>
          <property name="maximum" >
           <double>100000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleHeight" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum" >
           <double>100000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QComboBox" name="cmbWidthUnit" />
        </item>
        <item>
         <widget class="QComboBox" name="cmbHeightUnit" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblResolution" >
        <property name="text" >
         <string>&amp;Resolution:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleResolution" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="decimals" >
         <number>0</number>
        </property>
        <property name="maximum" >
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>191</width>
          <height>61</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpMode" >
     <property name="title" >
      <string>Mode</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="lblColorSpaces" >
        <property name="text" >
         <string>Color space:</string>
        </property>
        <property name="buddy" >
         <cstring>cmbColorSpaces</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KisCmbIDList" native="1" name="cmbColorSpaces" />
      </item>
      <item row="1" column="1" >
       <widget class="SqueezedComboBox" name="cmbProfile" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lblProfiles" >
        <property name="text" >
         <string>Profile:</string>
        </property>
        <property name="buddy" >
         <cstring>cmbProfile</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpContents" >
     <property name="title" >
      <string>Contents</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="1" >
       <widget class="QTextEdit" name="txtDescription" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="lblColor" >
        <property name="text" >
         <string>Canvas color:</string>
        </property>
        <property name="buddy" >
         <cstring>cmbColor</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KColorCombo" native="1" name="cmbColor" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lblDescription" >
        <property name="text" >
         <string>Description:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignTop</set>
        </property>
        <property name="buddy" >
         <cstring>txtDescription</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lblOpacity" >
        <property name="text" >
         <string>Opacity:</string>
        </property>
        <property name="buddy" >
         <cstring>sliderOpacity</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QFrame" name="opacityPanel" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1" >
           <property name="text" >
            <string>Transparent</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderOpacity" >
           <property name="toolTip" >
            <string>Opacity of the background color</string>
           </property>
           <property name="whatsThis" >
            <string>Opacity of the background color.</string>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
           <property name="pageStep" >
            <number>10</number>
           </property>
           <property name="value" >
            <number>100</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickInterval" >
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel2" >
           <property name="text" >
            <string>Opaque</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>480</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_createButton" >
       <property name="text" >
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KisCmbIDList</class>
   <extends>QWidget</extends>
   <header>kis_cmb_idlist.h</header>
  </customwidget>
  <customwidget>
   <class>SqueezedComboBox</class>
   <extends>QComboBox</extends>
   <header>squeezedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QWidget</extends>
   <header>kcolorcombo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>cmbColorSpaces</tabstop>
  <tabstop>cmbProfile</tabstop>
  <tabstop>cmbColor</tabstop>
  <tabstop>sliderOpacity</tabstop>
  <tabstop>txtDescription</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
